### ISSUE TITLE
Incompatibility with Pillow v6.x

### ISSUE THREAD

**Issue by hugorodgerbrown** [Distrotech/reportlab#10]
The `Image.VERSION` attribute was removed in Pillow from version 6 on (https://github.com/python-pillow/Pillow/commit/ef9bf76ce836d04630c48122cb828eba751096f0). This causes errors wherever it is calle: https://github.com/Distrotech/reportlab/blob/48cafb6d64ff92fd9d4f9a4dd888be6f7d55b765/src/reportlab/lib/utils.py#L867-L871

Intermediate fix is to use `PILLOW_VERSION` instead, although this will also be removed in future versions.

**Comment by datadidit** [comment]
We just ran into this as well had to hardcode the pillow version in the requirements.txt to get it to work  as a work around

```
Pillow==5.4.1
```

This needs have a top level req https://github.com/Distrotech/reportlab/blob/master/setup.py#L541 or  code needs to be updated to not be dependent on getting VERSION.

**Comment by hugorodgerbrown** [comment]
It looks like issues are only being responded to on Bitbucket?

**Comment by datadidit** [comment]
@hugorodgerbrown yeah several people have brought it up on their bitbucket too. Given they just did a release recently I'm hoping they push out a fix soon.

Installing Pillow after reportlab is a temporary patch.

**Comment by alexey-v-paramonov** [comment]
Patch:

https://github.com/gentoo/gentoo/blob/master/dev-python/reportlab/files/reportlab-3.5.13-pillow-VERSION.patch
